components/dropdown/dropdown-content-mixin.js

```diff
@@ -575,6 +575,15 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 
 		if (newValue) {
 
+			const evt = new CustomEvent('d2l-dropdown-before-open', {
+				bubbles: true,
+				composed: true,
```

Comment:
I'm guessing this needs to bubble and be composed because the event is originating from the dropdown content by the typical wire-up will be on dropdown?